lib/modes/ingest.ex

```diff
@@ -0,0 +1,55 @@
+defmodule ElixirSonicClient.Modes.Ingest do
+  alias ElixirSonicClient.TcpConnection
+
+  @default_bucket_name "default_bucket"
+
+  def push(conn, collection, object, term) do
+    TcpConnection.send(
```

Comment:
What do you think about a function that sends the command and return the response, this way we don't need to call send and recv all the time. We can have a module with some functions that we're going to use in multiple modules then we just include it on them.

Reply:
Yeah, this makes sense and considered this, but since this involved changing the code in many different places I thought that it would be beyond the scope of what we wanted to achieve in this task. What do you think if I create a task for this and move onto it as soon as we merge this PR?

Reply:
Yeah it makes sense.

Reply:
Cool, @edgarlatorre